src/components/AllArticles.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { getAllArticles } from "../api-calls/api-calls";
+import { ArticleCard } from "./AllArticles-components/ArticleCard";
+
+function AllArticles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    const params = {
+      //   sort_by: "title",
```

Comment:
don't forget this object needs a key of params that takes the object using your sort by
`const params = {params:{sort_by:'date'}`